docs/imagecustomizer/api/cosi.md

```diff
@@ -14,101 +21,110 @@ The COSI file MUST be an uncompressed tarball. The file extension SHOULD be `.co
 The tarball MUST contain the following files:
 
 - `metadata.json`: A JSON file that contains the metadata of the COSI file.
-- Partition image files in the folder `images`: The actual partition images
-  that will be used to install the OS.
+- Filesystem image files in the folder `images/`: The actual filesystem images
+  that Trident will use to install the OS.
+
+To allow for future extensions, the tarball MAY contain other files, but Trident
```

Comment:
When adding this file to this repo, some of the contents were changed to make the file more generic. For example, using the terms "writer" and "reader" instead of specific product names.